applications/GeoMechanicsApplication/custom_strategies/schemes/geomechanics_time_integration_scheme.hpp

```diff
@@ -11,14 +11,13 @@
 //
 #pragma once
 
-#include "solving_strategies/schemes/scheme.h"
 #include "geo_mechanics_application_variables.h"
+#include "solving_strategies/schemes/scheme.h"
 
 namespace Kratos
 {
 
-struct FirstOrderScalarVariable
-{
+struct FirstOrderScalarVariable {
```

Comment:
I'll have a look in the next format PR if there is a convenient way to adopt the same style for structs as for classes instead of following the style for if-statements and for loops